README.md

```diff
@@ -1 +1,59 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
 # Apache NiFi Standard Libraries
+
+A collection of standard implementations used across the NiFi ecosystem.
+
+## Table of Contents
+
+- [Getting Started](#getting-started)
+- [License](#license)
+
+## Getting Started
+
+### Requirements
+    
```

Comment:
Should this be Java 1.8 and newer now?

Reply:
I think for a while these libraries will have to be built using JDK 8 and should run on newer versions. We should probably add some information here:

JDK 1.8 for building.
JRE >= 1.8 for running or consuming these libraries.

Reply:
Will update